source/js/front/like.js

```diff
@@ -119,11 +78,13 @@ class Like {
 					[...mutation.addedNodes].forEach((node) => {
 						if (
 							node.nodeType === Node.ELEMENT_NODE &&
-							node.querySelector('.like-icon') &&
-							!node.querySelector('.like-icon').hasAttribute('data-js-has-click')
+							node.querySelector('.like-icon')
```

Comment:
Store the selector as a variable. Also reccurs further down.
```suggestion
							node.querySelector(likeIconSelector)
```